operations/conman/Troubleshoot_ConMan_Reset_Console_Services.md

```diff
@@ -0,0 +1,274 @@
+# Complete Reset of the Console Services
+
+At times the easiest way to get console services back up and running is to do a complete
+reset of the services. There is no persistent state so there is no backup/restore operation
+needed.
+
+> **`NOTE`** The console connections to all nodes will be disrupted for the duration of this
+procedure. Any active console sessions will be terminated and no console logging will occur.
+The existing console log files will be retained, but there will be a gap in the log file coverage.
+
+1. (`ncn-mw#`) Find the `cray-console-operator` pod.
+
+    The `cray-console-operator` pod controls the other pods in the console services. This pod will be
+    used to control the other pods.
+
+    ```bash
+    OP_POD=$(kubectl get pods -n services \
+            -o wide|grep cray-console-operator|awk '{print $1}')
+    echo $OP_POD
+    ```
+
+    Example output:
+
+    ```text
+    cray-console-operator-6cf89ff566-kfnjr
+    ```
+
+1. (`ncn-mw#`) Suspend the `cray-console-operator` and clear the current data.
+
+    1. Exec into the `cray-console-operator` pod.
+
+        ```bash
+        kubectl -n services exec -it $OP_POD -c cray-console-operator -- sh
+        ```
+
+    1. Source the helper function file.
+
+        ```bash
+        source /app/bashrc
+        ```
+
+    1. Suspend the current updates.
+
+        ```bash
+        suspend
+        ```
+
+    1. Clear the current status data in the database.
+
+        ```bash
+        clearData
+        ```
+
+    1. Exit from the `cray-console-operator` pod.
```

Comment:
```suggestion
1. Suspend the `cray-console-operator` and clear the current data.

    1. (`ncn-mw#`) Exec into the `cray-console-operator` pod.

        ```bash
        kubectl -n services exec -it $OP_POD -c cray-console-operator -- sh
        ```

    1. (`pod#`) Source the helper function file.

        ```bash
        source /app/bashrc
        ```

    1. (`pod#`) Suspend the current updates.

        ```bash
        suspend
        ```

    1. (`pod#`) Clear the current status data in the database.

        ```bash
        clearData
        ```

    1. (`pod#`) Exit from the `cray-console-operator` pod.
```